jetty-hazelcast/src/main/config/etc/sessions/hazelcast/remote.xml

```diff
@@ -10,21 +10,11 @@
   <Call name="addBean">
     <Arg>
       <New id="sessionDataStoreFactory" class="org.eclipse.jetty.hazelcast.session.HazelcastSessionDataStoreFactory">
-        <Set name="mapName">
-          <Property name="jetty.session.hazelcast.mapName" default="jetty-distributed-session-map" />
-        </Set>
-        <Set name="hazelcastInstanceName">
-          <Property name="jetty.session.hazelcast.hazelcastInstanceName" default="JETTY_DISTRIBUTED_SESSION_INSTANCE" />
-        </Set>
-        <Set name="useQueries">
-          <Property name="jetty.session.hazelcast.useQueries" default="false" />
-        </Set>
-        <Set name="gracePeriodSec">
-          <Property name="jetty.session.gracePeriod.seconds" default="3600" />
-        </Set>
-        <Set name="savePeriodSec">
-          <Property name="jetty.session.savePeriod.seconds" default="0" />
-        </Set>
+        <Set name="mapName" property="jetty.session.hazelcast.mapName"/>
+        <Set name="hazelcastInstanceName" property="jetty.session.hazelcast.hazelcastInstanceName"/>
+        <Set name="useQueries" property="jetty.session.hazelcast.useQueries"/>
+        <Set name="gracePeriodSec" property="jetty.session.gracePeriod.seconds"/>
+        <Set name="savePeriodSec" property="jetty.session.savePeriod.seconds"/>
         <Set name="onlyClient">
           <Property name="jetty.session.hazelcast.onlyClient" default="true" />
         </Set>
```

Comment:
Why was this kept as `Property`? Is it correct that embedded sees `false`, but standalone server sees `true`?

Reply:
This is a `Property` because we are setting the `default` value. As the same HazelcastSessionDataStoreFactory is used in both the `remote.xml `and `default.xml`, we cannot default to the value in the class, but must set this field explicitly to match either the remote or embedded case.  The other alternative is to make it a `property`, and remove the default and trust that the value in the .`mod/.ini `file is not commented out/correct.